feature-utils/poly-look/package.json

```diff
@@ -35,6 +34,7 @@
     "@testing-library/react": "^12.1.3",
     "@web/test-runner": "^0.13.22",
     "babel-jest": "^27.5.1",
+    "d3": "^7.6.1",
```

Comment:
This is just a change of place by nom I, but it fixed the version that was actually installed in package-lock.json. Better that way.